rpcd/playbooks/roles/rpc_maas/tasks/maas_exclude.yml

```diff
@@ -13,18 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Ensure local checks are removed
-  file:
-    path: "/etc/rackspace-monitoring-agent.conf.d/{{ item }}--{{ inventory_hostname }}.yaml"
-    state: absent
-  with_items:
-    - "{{ maas_excluded_checks }}"
-  delegate_to: "{{ physical_host }}"
```

Comment:
But this new task is what is gathering the list of files, right?  And that needs to run on the physical_host.